fuse_tutorials/launch/fuse_3d_tutorial.launch.py

```diff
@@ -0,0 +1,67 @@
+#! /usr/bin/env python3
+
+# Copyright 2022 Open Source Robotics Foundation, Inc.
```

Comment:
Do we change the copyright?

Reply:
Good catch, I did this upstream but forgot to also do it here.